src/explore-education-statistics-common/src/components/TabsSection.tsx

```diff
@@ -56,14 +59,16 @@ const TabsSection = forwardRef<HTMLElement, TabsSectionProps>(
 
     return hasRendered ? (
       <section
-        aria-labelledby={onMedia(tabProps['aria-labelledby'])}
+        aria-labelledby={
+          !isSingleTab ? onMedia(tabProps['aria-labelledby']) : ''
+        }
         hidden={hidden}
         className={classNames(classes.panel, styles.panel, {
           [styles.panelHidden]: hidden,
         })}
         id={id}
         ref={ref}
-        role={onMedia('tabpanel')}
+        role={!isSingleTab ? onMedia('tabpanel') : ''}
```

Comment:
would suggest making the single tab branch use `undefined` to prevent the `role` attribute appearing in the markup as it would just be empty.